### DISCUSSION TITLE
Feature request: Monitor Traffic by binary

### DISCUSSION THREAD

**Discussion by securegh** [GyulyVGC/sniffnet#597] [Ideas]
Hello, 

I really like your app, although it would be useful to know what application generated the traffic. Something like Glasswire, portmaster are doing. 

This feature would add great utility to your app, thank you for considering it!

**Comment by GyulyVGC** [comment]
Hey @securegh, thanks for your feedback and interest!

The feature you're requesting is already being discussed in issue #170 and is in the project's roadmap.
